phrosty/plotting.py

```diff
@@ -20,7 +20,9 @@
 
 roman_bands = ['R062', 'Z087', 'Y106', 'J129', 'H158', 'F184', 'W146', 'K213']
 
-def showimage(path=None,band=None,pointing=None,sca=None,xycoords=None,box=False,boxsize=1000,data_ext=1,cmap='Greys', **kwargs):
+def showimage(path=None, band=None, pointing=None, sca=None,
+              xycoords=None, box=False, boxsize=1000, data_ext=1,
+              cmap='Greys', return_fig=False, **kwargs):
     """
     Quickly display an original RomanDESC image from either a filepath or a 
     filter, pointing, and SCA ID. 
```

Comment:
Can you add documentation for the `return_fig` keyword and how it affects what is displayed or returned?